client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -107,6 +99,38 @@ export const PatternLibrary = ( {
 		},
 	} );
 
+	const isLoggedIn = useSelector( isUserLoggedIn );
+	const isDevAccount = useSelector( ( state ) => getUserSetting( state, 'is_dev_account' ) );
+
+	const handleViewChange = useCallback(
+		( view: 'grid' | 'list' ) => {
```

Comment:
```suggestion
	const handleViewChange = 
		( view: 'grid' | 'list' ) => {
```
I would avoid `useCallback` here. There are no functional requirements for using it, and I think it's doubtful it improves performance. This is an interesting article on the topic https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
Good POV. I'll drop that hook.